clang/include/clang/Basic/AttributeCommonInfo.h

```diff
@@ -192,6 +192,13 @@ class AttributeCommonInfo {
 
   bool isC23Attribute() const { return SyntaxUsed == AS_C23; }
 
+  bool isAlignas() const {
+    // In the current state of code, IsAlignas is only configured to return
```

Comment:
This comment is immediately stale once we commit this, right?  Also, it refers to `IsAlignas`, but the function name is `isAlignas`.

Reply:
This refers to the variable `IsAlignAs` (not the parent function `isAlignas`), and the sentence holds for the current state of code. I believe I'm trying to explain why I cannot use `IsAlignas`'s current behaviour here (as it is more invasive). Normally I'd expect this function to be a `const` accessor to a private member per programming conventions, which is not the case here, hence a comment.

I'm sensing the review comment confusion indicates I need to communicate this better in source comments. Suggestions welcome.

Reply:
How about:
```
// FIXME: In the current state, the IsAlignas member variable is only true with the C++
// `alignas` keyword but not `_Alignas`. The following expression works around the
// otherwise lost information so it will return true for `alignas` or `_Alignas` while still
// returning false for things like `__attribute__((aligned))`.
```